wordpress-mobile/WordPress-iOS
# Feature/9784 route activity log app

Fixes #9783 

Part of #9748 

This PR extracts the navigation logic from `NotificationDetailsViewController` into a `Router`, and uses it on `ActivityDetailViewController` to present native elements to display: 

- Posts.
- Comments.
- Internal WebView for anything else.

![activity_nav](https://user-images.githubusercontent.com/9772967/42786131-ff2d5560-8922-11e8-9dda-8f2dc44d8226.gif)

To test:

The easiest way to test is finding a comment event, since it has both a comment range and a post range:
- Find a comment event and open it.
- Tap con the `Comment` link.
- Check that it opens the corresponding internal Comments component.
- Go back and search for an event with a post link.
- Tap on the `Post` link.
- Check that it opens the corresponding internal Post component.
